src/manifolds/VectorBundle.jl

```diff
@@ -472,6 +472,11 @@ function get_vectors(M::VectorBundleFibers, p, B::CachedBasis)
 end
 Base.@propagate_inbounds Base.getindex(x::FVector, i) = getindex(x.data, i)
 
+function Base.getproperty(TpM::TangentSpaceAtPoint, sym::Symbol)
```

Comment:
Extending `getproperty` can cause serious type instability. Theoretically this method could be inlined and `sym` constant-propagated but it may not happen in practice.

Reply:
My hope was to have the `TangentSpaceAtPoint` with easy accessors for point and manifold (`TpM.fiber.manifold` feel weird here).

Reply:
What would you propose to do to avoid the `getproperty` but still get a nice interface?

Reply:
We can always extend `base_manifold` to `TangentSpaceAtPoint` to return `TpM.fiber.manifold`, I don't have a better idea.

Reply:
good idea.